### DISCUSSION TITLE
ignore-space-change (-b)?

### DISCUSSION THREAD

**Discussion by J-Kappes** [jesseduffield/lazygit#2626] [Q&A] [answered]
Is it (already) possible to view diffs with the -b flag in lazygit? I couldn't find the option if it exists.

**Comment by mark2185** [comment, marked as answer]
From the help menu (`?`):
```
ctrl+w    Toggle whether or not whitespace changes are shown in the diff view
```

**Comment by stefanhaller** [comment]
To be precise though, the ctrl-w function does not use the `-b` option (`--ignore-space-change`), but `-w` (`--ignore-all-space`). @J-Kappes I would be interested if that's a problem for you, or if you would like to choose between these.

**Reply by J-Kappes** [reply]
No, for my purposes that's just as good.
